Makefile

```diff
@@ -79,7 +79,7 @@ destroy_cluster: Vagrantfile
 	vagrant destroy -f;                                                  \
 	sed -ie '/# VAGRANT START/,/# VAGRANT END/d' ~/.ssh/config;          \
 	sed -ie '/IML Vagrant cluster/d' ~/.ssh/authorized_keys;             \
-	if [ ! -z "$$LIBVIRT" ]; then                                        \
+	if [ -n "$$LIBVIRT" ]; then                                          \
```

Comment:
Looks like you have some trailing whitespace here, if you end up needing to push again.